bindings/nodejs/__test__/fs.spec.mjs

```diff
@@ -14,7 +14,7 @@
 
 
 import test from 'ava'
-import { Operator, Fs } from '../index.js'
+import { Operator, ServiceType } from '../index.js'
```

Comment:
We already have `Scheme`, we can export (wrap) it directly.

Reply:
We can't use an enum wrapper directly on `napi-rs`. I copied it.